mne/evoked.py

```diff
@@ -531,10 +534,11 @@ def plot_topo(self, layout=None, layout_scale=0.945, color=None,
                                 vline=vline, fig_facecolor=fig_facecolor,
                                 fig_background=fig_background,
                                 axis_facecolor=axis_facecolor,
-                                font_color=font_color, show=show)
+                                font_color=font_color, merge_grads=merge_grads,
+                                show=show)
 
     def plot_topomap(self, times="auto", ch_type=None, layout=None, vmin=None,
-                     vmax=None, cmap='RdBu_r', sensors=True, colorbar=True,
+                     vmax=None, cmap=None, sensors=True, colorbar=True,
```

Comment:
I had to change ``cmap=None`` to make it work. The behavior should be identical to what it was before (apart from colorbar having correct range), so deprecation is not needed.